test/integration/lib/webpush_server.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const express = require('express');
+const Config = require('config-js');
+const getRawBody = require('raw-body');
+const assert = require('assert');
+const crypto = require('crypto');
+const ece = require('http_ece');
+const typer = require('media-typer');
+var config = new Config('./test/integration/lib/config/foxbox.js');
+
+var webpush_server = (function() {
+  var serverECDH = crypto.createECDH('prime256v1');
+  var serverPublicKey = serverECDH.generateKeys('base64');
+  var _server = express();
+  
+  var port = config.get('webpush.port');
+  var endpoint = config.get('webpush.endpoint');
+  
+  var incomingPushMsg;
+  var serverInstance;
+
+  function getEndpointURI() {
```

Comment:
but I don't think it works this case, because I am in an anonymous function, and not in a var.